subsys/net/lib/nrf_cloud/src/nrf_cloud_codec.c

```diff
@@ -19,6 +19,10 @@
 #include <modem/modem_info.h>
 #include "cJSON_os.h"
 
+#ifndef EFTYPE
+#define EFTYPE EINVAL
```

Comment:
```suggestion
#define EFTYPE EPROTO
```
is EPROTO available?  
EINVAL is already used as an error return value in the same functions as EFTYPE.

Reply:
I just replaced the occurrences with EPROTO. :+1: